LinaSaysHello/Program.cs

```diff
@@ -13,16 +13,54 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            //Console.WriteLine("Hi! What’s your name?");
-            //string name = Console.ReadLine();
-            //Console.WriteLine("Nice to meet you " + name);
-            //Console.WriteLine("Anastasiya");
-            //Console.WriteLine("Vention");
-            //Console.ReadLine();
+            //Task1//
+            int num = 20;
+            num += 5;
 
-           
+            Console.WriteLine("Variable:" + num);
 
+            //Task2//
 
+            Console.Write("Enter a 4 digit number of days: ");
+            int number = Convert.ToInt32(Console.ReadLine());
+            int years = number / 365;
+            int months = (number - years * 365) / 30;
+            int days = (number - years * 365) - months * 30;
+
+            Console.WriteLine("This number represents: " + years + " years " + months + " months " + days + " days");
```

Comment:
Small useful recommendation for future: 
For constants such as the number of days in a month or a year, you should create variables. Because, for example, I will say "Lina, please calculate the number of months in a four-digit number, taking into account that there are 31 days in a month". And you'll have to look for all uses of the number 30 to change it to 31. 
And in the case of using a variable for a constant, you only have to change its value :)